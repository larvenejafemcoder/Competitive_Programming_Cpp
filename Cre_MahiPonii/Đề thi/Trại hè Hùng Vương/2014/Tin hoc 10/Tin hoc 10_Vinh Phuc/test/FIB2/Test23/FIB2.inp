32
24 63285
36 11749829
35 1334035
17 1072
37 36062135
42 15800766
3 2
26 165155
26 135185
29 740390
2 2
15 908
26 103623
21 206
4 1
3 3
40 49491737
44 819897348
38 13031213
5 4
37 10708278
30 211077
31 1808849
26 172988
43 555375660
41 236370095
40 128743146
12 144
4 3
27 116031
23 25526
26 134876
