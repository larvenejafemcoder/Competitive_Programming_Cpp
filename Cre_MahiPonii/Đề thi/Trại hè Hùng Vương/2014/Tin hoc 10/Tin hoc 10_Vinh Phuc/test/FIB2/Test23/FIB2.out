24173
4488035
509556
410
13774510
6035356
1
63084
51636
282804
1
347
39581
79
1
1
18904161
313172920
4977481
2
4090198
80624
690919
66076
212134626
90285342
49175506
55
1
44320
9750
51518
